_drafts/_2023-04-14-memfault-matter.md

```diff
@@ -0,0 +1,358 @@
+---
+title: Matter, Thread, and Memfault
+description:
+  Matter and Thread are changing the smart home landscape with IPv6-based
+  connectivity. This post shows how to use Memfault in a matter/Thread
+  environment - the changes required to the Memfault SDK, the required set-up
+  and ideas for future integration of Memfault with Thread.
+author: mabe
+image: img/<post-slug>/cover.png # 1200x630
+---
+
+<!-- excerpt start -->
+
+I"m Markus, software engineer @ [Tridonic](https://tridonic.com), where we are working on [Internet-connected wireless lighting solutions](http://www.tridonic.com/matter) based on the [matter standard](https://csa-iot.org/all-solutions/matter/). To be able to monitor the reliability of those devices we've been using Memfault and tied it into matter/Thread and its UDP/IPv6 stack based on the Nordic Connect SDK. In the following, I'll show you the modifications we've done to enable Memfault in an IPv6 solution.
+
+<!-- excerpt end -->
+
+{% include newsletter.html %}
+
+{% include toc.html %}
+
+## matter & Thread
+
+matter <https://csa-iot.org/all-solutions/matter/> is an interoperability standard for the IoT from the [connectivity standards alliance (csa)](https://csa-iot.org) previously known as the zigbeeAlliance. The matter standard is implemented in the open-source reference implementation in the [GitHub project-chip repository](https://github.com/project-chip/connectedhomeip/). matter currently uses UDP on top of IP(v6) and can run on top of different lower layers: Ethernet, WiFi, Thread/802.15.4. [Thread](https://www.threadgroup.org/) is a low-power IPv6 connectivity standard. Its reference implementation is available in the [openThread Github repository](https://github.com/openthread/openthread). The matter SDK as well as the openThread SDK are also available bundled in the [Nordic Connect SDK](https://developer.nordicsemi.com/nRF_Connect_SDK/).
```

Comment:
Not asking for any information in particular. Really it would just be nice to frame the benefits of using Matter or Thread. This paragraph is very descriptive, but I would like to hear your thoughts on why it's good, and why people should use it.

Reply:
added a paragraph on what we think are the benefits of matter and thread.